types/applications/sidebarTabs/sidebarDirectories/sceneDirectory.d.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * A directory listing of active game scenes
+ */
+declare class SceneDirectory extends SidebarDirectory {
+  /**
+   * @override
+   */
+  static get entity(): string;
+
+  /**
+   * @override
+   */
+  static get collection(): Scenes;
+
+  /**
+   * @override
+   * @remarks This is never called.
+   */
+  protected _onCreate(event: MouseEvent): Promise<Scene>;
```

Comment:
For the other `_onCreate` methods, you typed the parameter to be of type `Event`. As this seems to never be called, it probably doesn't matter, but it should be consistent.